webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/hooks/RunNuCommand.scala

```diff
@@ -97,7 +99,7 @@ object RunNuCommand {
       BasePosixProcess.LOGGER.setUseParentHandlers(false)
       val h = new ConsoleHandler()
       h.setFormatter(new SimpleFormatter() {
-        private val format = "%1$tFT%1$tT%1$tz %2$-7s %3$s: %4$s"
+        private val format = "[%1$tFT%1$tT%1$tz] %2$-7s %3$s: %4$s\n"
```

Comment:
this is to better mimic our logger format when an error happen inside nu command